e2e/testcafe-devextreme/tests/accessibility/menu.ts

```diff
@@ -0,0 +1,78 @@
+import { Item, Properties } from 'devextreme/ui/menu.d';
+import Menu from 'devextreme-testcafe-models/menu';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const items: Item[] = [
+  {
+    text: 'remove',
+    icon: 'remove',
+    items: [
+      {
+        text: 'user',
+        icon: 'user',
+        disabled: true,
+        items: [{
+          text: 'user_1',
+        }],
+      },
+      {
+        text: 'save',
+        icon: 'save',
+        items: [
+          { text: 'export', icon: 'export' },
+          { text: 'edit', icon: 'edit' },
+        ],
+      },
+    ],
+  },
+  { text: 'user', icon: 'user' },
+  {
+    text: 'coffee',
+    icon: 'coffee',
+    disabled: true,
+  },
+];
+
+const options: Options<Properties> = {
+  items: [items],
+  disabled: [true, false],
+  hint: [undefined, 'hint'],
+  orientation: ['horizontal', 'vertical'],
+  selectionMode: ['single', 'none'],
+  submenuDirection: ['auto', 'leftOrTop', 'rightOrBottom'],
```

Comment:
The same: https://github.com/DevExpress/DevExtreme/pull/27357#discussion_r1599405273